myScript.js

```diff
@@ -39,4 +39,149 @@ function createList() {
   };
 }
 
-document.getElementById('humberger-icon').onclick = createList; // modify
+document.getElementById('humberger-icon').onclick = createList;
+
+/// /////////////////////////////////////////////////////////////////////
+
+const projects = [
+  {
+    title: 'Multi Post Stories',
+    discription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featureImg: './img/card1-project-img.png',
+    technologies: ['html', 'bootstrap', 'ruby on rail'],
+    linkToLive: 'https://github.com/',
+    linkToSource: 'https://github.com/yasinabdmahmood/portfolio-website',
```

Comment:
- PLease I notice that your popup always redirects to your GitHub page after it is displayed for 4 to 5 seconds, please try fixing that.
- Please try to make all cards have a popup, because currently the first project card does not have a popup it just navigates to the nav bar on the page.
- [Optional] **Recommended** Please I noticed while I clicked on the `see project` it overlapped the `popup card` until I take my mouse off, please try fixing it, you can try `z-index CSS property`

##### Below is a video of the effect mentioned above

https://user-images.githubusercontent.com/58771507/181102416-56952dbf-ace5-4c54-81a5-4b39c25d700b.mp4